1.9/overview/architecture/index.md

```diff
@@ -0,0 +1,29 @@
+---
+post_title: Architecture
+nav_title: Architecture
+menu_order: 2
+---
+
+DC/OS is a software platform on which user software runs. This platform layer runs on top of the infrastructure layer. The infrastructure layer is composed of virtual or physical hardware including compute, storage, and networking.
+
+![DC/OS Architecture Layers](/docs/1.9/overview/architecture/img/dcos-architecture-layers.png)
+
+## Software Layer
+
+At the software layer, DC/OS provides package management and a package repository to easily install and manage multiple types of services: databases, message queues, stream processors, artifact repositories, monitoring solutions, continuous integration tools, source control management, log aggregators, etc. In addition to these packaged apps and services, the user may install their own custom apps, services, and scheduled jobs. For more information, see [Primitives](/docs/1.9/overview/architecture/primitives/).
```

Comment:
Suggest: `In addition to these packaged apps and services, you can install your own custom apps...`